src/app/helptext/system/reporting.ts

```diff
@@ -11,16 +11,16 @@ graphite_placeholder: T('Remote Graphite Server Hostname'),
 graphite_tooltip: T('Hostname or IP address of a remote  \
  <a href="http://graphiteapp.org/" target="_blank">Graphite</a> server.'),
 
-graph_age_placeholder: T('Graph Age'),
-graph_age_tooltip: T('Maximum time a graph is stored in months. \
+graph_age_placeholder: T('Graph Age in Months'),
+graph_age_tooltip: T('Maximum time a graph is stored in months (allowed values are 1-60). \
  Changing this value causes the <i>Confirm RRD Destroy</i> \
  checkbox to appear. Changes do not take effect until the existing \
  reporting database is destroyed.'),
 graph_age_validation: [regexValidator(/^\d+$/), rangeValidator(1, 60 )  ],
 
-graph_points_placeholder: T('Graph Points'),
+graph_points_placeholder: T('Number of Graph Points'),
 graph_points_tooltip: T('Number of points for each hourly, daily, \
- weekly, monthly, or yearly graph. Do not set this less than the \
+ weekly, monthly, or yearly graph (allowed values are 1-4096). Do not set this less than the \
```

Comment:
I think with determined that the "Do not set this less" sentence is obsolete and should be removed.  Maybe @dszidi can verify?